products/monitoring/api.yaml

```diff
@@ -855,4 +857,189 @@ objects:
         without removing the channel from all alerting policies that reference the channel.
         This is a more convenient approach when the change is temporary and you want
         to receive notifications from the same set of alerting policies on the channel
-        at some point in the future.
\ No newline at end of file
+        at some point in the future.
+  
+  
+  - !ruby/object:Api::Resource
+    name: UptimeCheckConfig
+    update_verb: :PATCH
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/monitoring/api/v3/'
```

Comment:
This should be documentation for the feature itself, not for the API. So https://cloud.google.com/monitoring/uptime-checks/ I think is the one to put here.